src/lib/testLogger.js

```diff
@@ -86,20 +85,15 @@ VerboseLogger.prototype.traverseTestResults = function(node, indentation) {
  * @param {string} indentation - Indentation used for formatting.
  */
 VerboseLogger.prototype.printTestTitles = function(testTitles, indentation) {
-  let prefixColor, statusPrefix;
+  let statusPrefix;
 
   for (let i = 0; i < testTitles.length; i++) {
     if (testTitles[i].failureMessages.length === 0) {
-      prefixColor = colors.GREEN;
-      statusPrefix = this._config.noHighlight ? 'PASS - ' : '\u2713 ';
```

Comment:
Based on your other comment, I think you see that it's "no" for right here, but we still print that for the file summary.
<img width="297" alt="screen shot 2015-12-09 at 8 12 22 pm" src="https://cloud.githubusercontent.com/assets/8445/11706841/7ef8f8c6-9eb1-11e5-9f61-07b06f787044.png">